### DISCUSSION TITLE
links do not work on Touch devices

### DISCUSSION THREAD

**Discussion by VirtualConferenceHubs** [mozilla/hubs#2945] [General Questions & Discussion] [answered]
Hi
On touch devices we are not able to pause/play a video or click on a external link (link to a website/google slide).
Is that a fault or its by design ?

Those links work perfectly fine on Laptop.

-Anubhav

**Comment by emclaren** [comment, marked as answer]
Hi @VirtualConferenceHubs  - thanks for letting us know. If this is on iOS devices (iphones/ipads) this issue is tracked here:  https://github.com/mozilla/hubs/issues/2163

There is a work around described there, but we need to find a better way to surface this to people.

If you are using an Android device, please let us know because this would be a different issue.

**Reply by emclaren** [reply]
Also, if you try the work around described in the linked issue #2163  let us know if it works or not. This will be helpful for troubleshooting this bug!
